clients/cobol-lsp-vscode-extension/src/__tests__/services/copybook/CopybookResolveURITest.spec.ts

```diff
@@ -32,6 +33,7 @@ const RELATIVE_CPY_FOLDER_NAME = "../relativeCobcopy";
 const folderPath = path.join(__dirname, CPY_FOLDER_NAME);
 
 jest.mock("vscode", () => {
+  // eslint-disable-next-line @typescript-eslint/no-require-imports
   const Uri = require("../../../__mocks__/UriMock").Uri;
```

Comment:
I think we should be able to just import it.

Reply:
I tried that but the tests fail then. As this whole test suite is IMHO quite bad, I would just leave the import as it is and focus on the whole module later.